src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkProfileTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// <auto-generated>
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+// 
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+// 
+// For documentation on code generator please visit
+//   https://aka.ms/nrp-code-generation
+// Please contact sdnnrp@microsoft.com if you need to make changes to this file.
+// </auto-generated>
+
+using System;
+using Microsoft.Azure.ServiceManagemenet.Common.Models;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Commands.Network.Test.ScenarioTests
+{
+    public class NetworkProfileTestsGenerated : RMTestBase
```

Comment:
you need to checkin the test recordings for all these tests.

Reply:
yep, these will be in a subsequent iteration.